cmd/down.go

```diff
@@ -73,3 +82,16 @@ func init() {
 	RootCmd.AddCommand(downCmd)
 	downCmd.Flags().Int64("grace-period", -1, "Number of seconds given to resources to terminate gracefully. A negative value is ignored.")
 }
+
+func deleteMongoDBSecret(c kubecfg.DeleteCmd, name, ns string) error {
```

Comment:
you can call this function deleteSecret (it doesn't have anything related to mongoDB)